### DISCUSSION TITLE
Informationen zur Betreiberverordnung anzeigen

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#75] [Shipped - Live on production]
Im gesamten Bereich der Einweisungen ist die Betreiberverordnung (Anlage 1, 2, 1+2, keine) eine elementare Information. User wissen häufig nicht, welche Geräte ein entsprechendes Risiko haben bzw. welche Geräte zu welcher Anlage gehören und entsprechend auch nicht ob eine Hersteller oder Anwendereinweisung ausreicht.

In folgenden Bereichen sollte daher die Betreibervordnung standardmäßig mit angezeigt werden.
1. Geräteauswahl neuer Vorgang
2. Geräteauswahl Gruppen
3. Geräteauswahl Matrix
4. Gerätedetails in der Matrix

**Comment by YvesR** [comment]
Das ist so ein "deutsches Ding", ich würde hier eher auf das Anwendungsrisiko gehen, das kann in DE dann ja 1:1 in Referenz stehen.

**Reply by moritzvolkmar** [reply]
Definitiv ein DE Ding. Liefert aber enorme Mehrwerte im Bereich Einweisungen. 
Wir nutzen das Anwendungsrisiko zwar als 1:1 zur Betreiberverordnung, aber geschult wird tatsächlich Anlage 1, 2, 1+2. Müssen wir nochmal genau besprechen und entscheiden.

**Reply by YvesR** [reply]
@moritzvolkmar Sollten wir also in DE only Anlage 1 Filter einbauen und bei allen anderen Anwendungsrisiko?
